cypress/fixtures/stubbing/ScenarioSharing/scenarios.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { DEFAULT_SCENARIOS_LIST } from '../default/scenarios';
+import { USER_EXAMPLE, USERS_LIST } from '../default/users';
+import { ROLES } from '../../../commons/constants/generic/TestConstants';
+
+export const UNSHARED_SCENARIOS_LIST = DEFAULT_SCENARIOS_LIST.map((scenario) => {
+  return {
+    ...scenario,
+    security: {
+      default: ROLES.SCENARIO.VIEWER,
+      accessControlList: [{ id: USER_EXAMPLE.email, role: ROLES.SCENARIO.ADMIN }],
```

Comment:
Not sure what `UNSHARED_SCENARIOS_LIST` stands for, but if you set the default role to "viewer", all users will be able to see this scenario

Reply:
I will set it to 'none'